Code/Tools/Libs/ModelImporter2/ImporterAssimp/AssimpUtils.cpp

```diff
@@ -0,0 +1,63 @@
+#include <ModelImporterPCH.h>
+
+#include <assimp/matrix4x4.h>
+#include <assimp/quaternion.h>
+#include <assimp/types.h>
+#include <assimp/vector3.h>
+
+namespace ezModelImporter2
+{
+  ezColor ConvertAssimpType(const aiColor4D& value, bool invert = false)
```

Comment:
VS complains here:
bool invert/* = false*/ instead

Reply:
Interesting, didn't complain for me.